parking_permits/services/traficom.py

```diff
@@ -149,9 +149,7 @@ def fetch_vehicle_details(self, registration_number, permit=None):
                     emission_type = EmissionType.WLTP
 
         mass = et.find(".//massa")
-        module_weight = mass.find("modulinKokonaismassa")
-        technical_weight = mass.find("teknSuurSallKokmassa")
-        weight_et = module_weight if module_weight is not None else technical_weight
+        weight_et = mass.find("omamassa")
```

Comment:
`weight_et` could be None so I would change this line to:

`weight = safe_cast(weight_et.text, int, 0) if weight_et and weight_et.text else 0`